Makefile

```diff
@@ -1,7 +1,7 @@
 # author: Sylvain Anderegg
 
 # TODO: add flavours by combinging docker-compose files. Namely development, test and production.
-VERSION := $(shell cat /proc/version)
+VERSION := $(shell uname -a)
```

Comment:
Was this also tested under Windows?

Reply:
no, but /proc/version is not even portable among unix OSs ... so ...